File: 1040.c
#include<stdio.h>
int main()
{
    float a, b, c, d, e;
    scanf("%f %f %f %f", &a, &b, &c, &d);
    float avg = (a*2 + b*3 + c*4 + d)/10.0;
    printf("Media: %.1f\n", avg);
    if(avg>=7.0)
        printf("Aluno aprovado.\n");
    else if(avg<5.0)
        printf("Aluno reprovado.\n");
    else if(avg>=5.0 && avg<=6.9)
    {
        printf("Aluno em exame.\n");
        scanf("%f", &e);
    printf("Nota do exame: %0.1f\n", e);
    float avg2= (avg+e)/2;
    if(avg2>=5.0)
        printf("Aluno aprovado.\n");
    else
        printf("Aluno reprovado.\n");
    printf("Media final: %0.1f\n", avg2);
    }



    return 0;
}

File: 1040.py
a, b, c, d= map(float, input().split())
avg = float((a*2 + b*3 + c*4 + d)/10.0)
print(f"Media: {avg:0.1f}")
if avg>=7.0:
    print("Aluno aprovado.")
elif avg<5.0:
    print("Aluno reprovado.")
elif (5.0<=avg<=6.9):
    print("Aluno em exame.")
    e = float(input())
    print(f'Nota do exame: {e:0.1f}')
    avg2= float(avg + e)/2
    if avg2>=5.0:
        print('Aluno aprovado.')
    else:
        print("Aluno reprovado.")
    print(f"Media final: {avg2:.1f}")
